src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -150,17 +150,27 @@ private void ProcessCatalogItems(IndexWriter indexWriter, IEnumerable<JObject> c
 
             foreach (JObject catalogItem in catalogItems)
             {
-                _logger.LogInformation("Process CatalogItem {CatalogItem}", catalogItem["@id"]);
+                _logger.LogInformation("Process CatalogItem {CatalogItem}", catalogItem["@id"].ToString());
```

Comment:
This was an existing bug where the log message would read:

    Process CatalogItem []

...instead of having the catalog @id URL inside the square brackets.